sale_order_line_discount_validation/report/sale_report.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, models
+from odoo.exceptions import Warning as Warn
```

Comment:
this change was made due [W0622(redefined-builtin), ] Redefining built-in 'Warning' pylint warning

Reply:
@xhnilic3 Warning is deprecated since a long time. Please use UserError

Reply:
@rousseldenis thank you for an advice, will do so

Reply:
done